demophin.py

```diff
@@ -117,6 +117,8 @@ def parse_sentence(grm, sent, n=None):
         for i in range(len(opts['cmdargs'])):
             if opts['cmdargs'][i].startswith('-n '):
                 opts['cmdargs'][i] = '-n ' + str(n)
+    #  uniqify
+    opts['cmdargs'] = list(set(opts['cmdargs']))
```

Comment:
Calling `set()` is a bad idea because argument names and values are sometimes different items in the list (as they should be; see demophin.json, which has `['-n', '5']`). Also, I think it's unnecessary, because with ACE (at least for the current version, I can't speak for future versions), the last argument value is used. E.g., the following uses the `jacy.dat` grammar and `-n 2`:

    ace -g erg.dat -g jacy.dat -n 1 -n 2